hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/EncryptedFileSystem.java

```diff
@@ -98,6 +100,23 @@ public boolean rename(Path src, Path dst) throws IOException {
         return renamed;
     }
 
+    @Override
+    public boolean delete(Path path, boolean recursive) throws IOException {
```

Comment:
I was trying to avoid deleting key material if a folder path is passed in for a recursive delete, but as noted in the other comment, it should be safe even if there's no key material.